CONTRIBUTING.md

```diff
@@ -117,8 +117,36 @@ Before you begin, ensure you have:
 #### Setup Steps
 
 1. Clone the repository
-2. Set up your local Kubernetes environment (TBD)
-3. Configure local Docker registry (TBD)
+2. Configure local Docker registry
```

Comment:
just an idea  - we can wrap steps for publishing images and charts (maybe something else) into make targets - to make it dumb simple for users.